marc_config.rb

```diff
@@ -39,7 +39,7 @@
 to_field 'cho_creator', extract_role('700', 'creator')
 to_field 'cho_creator', extract_role('710', 'creator')
 to_field 'cho_creator', extract_role('711', 'creator')
-to_field 'cho_date', marc_publication_date
+to_field 'cho_date', marc_publication_date, transform(&:to_s)
```

Comment:
this is also referenced in the README.  Can you change it there as well?

Reply:
Done.